index.html

```diff
@@ -0,0 +1,135 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=0.1">
+	<title>Ellie Robinson - Developer</title>
+	<link href="normalise.css" type="text/css" rel="stylesheet"> <!-- this makes sure all browsers start with the same style -->
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Source+Sans+Pro:wght@200;600&display=swap" rel="stylesheet">
+
+	<script src="https://kit.fontawesome.com/d8f8b9f853.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<section class="navigationBar">
+		<div>
+			<img class="navLogo" src="externalFiles/Ellie-Robinson2.png">
+		</div>
+		<nav>
+			<a href="#AboutMe">About Me</a>
+			<a href="#Portfolio">Portfolio</a>
+			<a href="#Contact">Contact</a>
+		</nav>
+	</section>
+	<header class="welcome">
+		<div class="intro">
+			<h1>Welcome!</h1>
+			<p>
+				I'm currently training to be a full stack developer at Mayden Academy.
+			</p>
+			<p>
+				Feel free to have a look around.
+			</p>
+		</div>
+		<div class="backgroundVideo">
+			<video preload autoplay muted playsinline loop>
+				<source src="externalFiles/production ID_4629650.mp4" type="video/mp4">
+				The Video is not playing.
```

Comment:
Sorry I should've been more specific, before you had written this:
"Your browser is not playing because the browser doesn't support it."
And I thought you probably meant to write this:
"The video is not playing because your browser doesn't support it."

If you wanted to keep your original message you could also say:
"You're missing out on a great video because your browser doesn't support it."

Hopefully that clears it up 😅